### PR TITLE
Popup Opening Sound

### PR DESCRIPTION
**Pull request by fpcorso**
Adds a new feature to allow admins to set a sound to play when the popup opens.

Related issue: #701

### CODE REVIEW

**Comment by fpcorso** [comment]
@danieliser, this is now ready to be reviewed and merged. The final feature matches what was outlined over in our issue (#701)

File: assets/js/src/site/plugins/pum.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ var PUM;
         default_theme: '0',
         home_url: '/',
         version: 1.7,
+		pm_dir_url: '',
```
Comment (danieliser) [inline assets/js/src/site/plugins/pum.js]: This remnants of your code formatting issues?

File: classes/Admin/Popups.php [hunk 2]
```diff
@@ -562,6 +563,34 @@ public static function fields() {
...
+							),
+						),
+						'custom_sound'   => array(
+							'label'    => __( 'Custom Sound URL', 'popup-maker' ),
+							'desc'     => __( 'Enter URL to sound file.', 'popup-maker' ),
+							'type'     => 'text',
+							'std'      => '',
+							'priority' => 10,
+							'dependencies' => array(
+								'open_sound' => array( 'custom' ),
```
Comment (danieliser) [inline classes/Admin/Popups.php]: Just an FYI, not that you need to change it, you can also simplify in this case to `'open_sound' => 'custom',`

Comment (fpcorso) [reply classes/Admin/Popups.php]: Ah, I just followed what some of the other fields were doing. We probably need to whip up some docs about our fields system at some point.

File: assets/js/src/site/plugins/pum.js (hunk 1 above)
Comment (fpcorso) [reply assets/js/src/site/plugins/pum.js]: This is what I was referring to over in Slack. My line follows our tab requirements. Two tabs. The lines around it are using 8 spaces.

**Pull request merged**